charts/brigade/templates/gateway-cr-deployment.yaml

```diff
@@ -32,6 +32,10 @@ spec:
         role: gateway
         type: dockerhub
     spec:
+      {{ if .Values.api.affinity }}
```

Comment:
These should be `.Values.cr.affinity`, right?

Reply:
correct, fixed